src/raw/mod.rs

```diff
@@ -297,35 +298,29 @@ impl<K, V> Table<K, V> {
     }
 
     #[inline]
-    pub(crate) fn bin<'g>(&'g self, i: usize, guard: &'g Guard) -> Shared<'g, BinEntry<K, V>> {
+    pub(crate) fn bin<'g>(&'g self, i: usize, guard: &'g Guard<'_>) -> Shared<'g, BinEntry<K, V>> {
         self.bins[i].load(Ordering::Acquire, guard)
     }
 
     #[inline]
     #[allow(clippy::type_complexity)]
-    pub(crate) fn cas_bin<'g, P>(
```

Comment:
I think the `P` here was to make it easier to stick in both shared and owned, but I suppose that's not terribly important to retain.